### PR TITLE
Swik 2046 zoom in edit mode

### PR DESCRIPTION
**Pull request by vkovtun**

### CODE REVIEW

**Comment by abijames** [comment]
works well @kadevgraaf ok to merge!

File: components/Deck/ContentPanel/SlideModes/SlideViewPanel/SlideContentView.js [hunk 1]
```diff
@@ -122,6 +83,7 @@ class SlideContentView extends React.Component {
         };
         const contentStyle = {
             minWidth: '100%',
+            height: '720px',
```
Comment (kadevgraaf) [inline components/Deck/ContentPanel/SlideModes/SlideViewPanel/SlideContentView.js]: This preset might prevent other slide dimensions from working properly

File: components/Deck/ContentPanel/SlideModes/SlideEditPanel/SlideContentEditor.js [hunk 2]
```diff
@@ -2155,60 +2160,39 @@ class SlideContentEditor extends React.Component {
...
             minWidth: '100%',
-            // maxHeight: 450,
-            //padding: 10,
-            /*paddingLeft: 50,
-            paddingRight: 50,
-            paddingTop: 10,
-            xpaddingBottom: 10,*/
-            minHeight: 610,
-            overflowY: 'auto',
+            height: '720px',
```
Comment (kadevgraaf) [inline components/Deck/ContentPanel/SlideModes/SlideEditPanel/SlideContentEditor.js]: This preset might prevent other slide dimensions from working properly

**Comment by kadevgraaf** [comment]
screen shot 2018-06-22 at 10 56 18
screen shot 2018-06-22 at 10 56 31

File: components/Deck/ContentPanel/SlideModes/SlideViewPanel/SlideContentView.js [hunk 3]
```diff
@@ -44,41 +38,10 @@ class SlideContentView extends React.Component {
...
-                $('.pptx2html').css({'transform': '', 'transform-origin': ''});
-                $('.pptx2html').css({'transform': 'scale('+this.scaleratio+','+this.scaleratio+')', 'transform-origin': 'top left'});
-
-                let pptxheight = $('.pptx2html').outerHeight();
-
-                const scrollbarHeight = this.refs.inlineContent.offsetHeight - this.refs.inlineContent.clientHeight;
-                this.refs.slideContentView.style.height = (pptxheight * this.scaleratio + scrollbarHeight) + 'px';
-
-                //$('.pptx2html').css({'borderStyle': 'double', 'borderColor': '#DA6619'});
-                $('.pptx2html').css({'borderStyle': 'double', 'borderColor': 'rgba(218,102,25,0.5)'});
```
Comment (kadevgraaf) [inline components/Deck/ContentPanel/SlideModes/SlideViewPanel/SlideContentView.js]: borderstyle is missing in slide edit view

**Comment by kadevgraaf** [comment]
Hmmm, this might also be partially a generic problem, see e.g.: https://slidewiki.org/deck/84941/slidewiki-guide-introduction/slide/585194-8 
Can you check @vkovtun ?

**Comment by vkovtun** [comment]
@kadevgraaf please check my changes.

**Comment by abijames** [comment]
@vkovtun can you check the conflicts on this branch and update it with master so I can test it again? Currently the CSS isn't loading due to changes made to master last week

**Comment by vkovtun** [comment]
@kadevgraaf, I have made the changes. Please check them.

**Comment by kadevgraaf** [comment]
Hi @vkovtun! 
Git says "This branch is 13 commits ahead, 265 commits behind master"  and there are merge conflicts in  components/Deck/ContentPanel/SlideModes/SlideEditPanel/SlideContentEditor.js
custom_modules/custom-semantic-ui
custom_modules/reveal.js 
Can you resolve these first?

**Comment by vkovtun** [comment]
@kadevgraaf, how can I resolve them? Should I do merging locally?

**Comment by kadevgraaf** [comment]
I'll try to fix them with submodule trickyness

**Comment by kadevgraaf** [comment]
Done 👍

**Comment by vkovtun** [comment]
@kadevgraaf check my new changes. Some slides in your example do produce that horizontal scroll bar, but the reason for that is deeper: the content overlaps the red border. I cannot fork your deck to check if editing helps, because forking is not working at the moment. For the same reason I could not check if I see the overflowing text in the edit mode.

**Comment by vkovtun** [comment]
Hi @kadevgraaf, can you please check my new changes?

**Comment by kadevgraaf** [comment]
Looking good! Many thanks!

**Pull request merged**